### DISCUSSION TITLE
Journald full of "overlayfs: "xino" feature enabled using 3 up"

### DISCUSSION THREAD

**Discussion by MrksHfmn** [opencontainers/runc#4506] [General]
After upgrading from 1.1.15-1 to 1.2.0-1 in Arch Linux, the journald log is full of the above message.
Was there any major change? Is it safe to ignore? Possible to mask the message in runc?

Topic about this issue: https://bbs.archlinux.org/viewtopic.php?id=300504

**Comment by cyphar** [comment]
It is safe to ignore, we use overlayfs to protect the runc binary. I guess on Arch Linux the `/run` directory is a separate filesystem to `/usr/bin` and so overlayfs outputs that log message (without rate limiting).

I think if we set `xino=off` for the overlayfs we should be able to silence that log message.

**Reply by MrksHfmn** [reply]
/run is mounted as tmpfs

`run /run tmpfs rw,nosuid,nodev,relatime,mode=755,inode64 0 0`

**Comment by cyphar** [comment]
https://github.com/opencontainers/runc/pull/4509 should fix the issue.
